File: src/main/java/com/wikia/tibia/objects/Loot.java
package com.wikia.tibia.objects;

import lombok.*;

import java.util.List;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor(access = AccessLevel.PUBLIC)
public class Loot {

    private String pageName;
    private String version;
    private String kills;
    private String name;
    private List<LootStatisticsItem> loot;

    public boolean isEmpty() {
        return loot == null ||
                loot.isEmpty() ||
                loot.size() == 1 && "Empty".equals(loot.get(0).getItemName());
    }
}

File: src/main/kotlin/com/wikia/tibia/objects/Loot.kt
package com.wikia.tibia.objects

data class Loot(
        val pageName: String,
        val version: String,
        val kills: String,
        val name: String,
        val loot: List<LootStatisticsItem>?
) {
    fun isEmpty() = loot == null ||
            loot.isEmpty() ||
            loot.size == 1 && "Empty" == loot[0].itemName
}